app/screens/receive-bitcoin-screen/receive-usd.tsx

```diff
@@ -145,6 +146,45 @@ const ReceiveUsd = () => {
   const { lnUpdate } = useSubscriptionUpdates()
   const [showMemoInput, setShowMemoInput] = useState(false)
   const [showAmountInput, setShowAmountInput] = useState(false)
+  const { timeLeft, startCountdownTimer, resetCountdownTimer } = useCountdownTimer()
+  const timeUntilInvoiceExpires = useMemo(() => {
+    if (invoice) {
+      return (
+        getLightningInvoiceExpiryTime(decodeInvoiceString(invoice.paymentRequest)) -
```

Comment:
This value could be negative, and your hook doesnt currently handle negative values.

Reply:
I guess in practice this could only happen if the app came back from background and the invoice had already expired - but you're right we should just set the invoice as expired if the value is negative and not start the timer.